File: app/src/main/java/com/jawnnypoo/openmeh/github/Contributor.java
package com.jawnnypoo.openmeh.github;

import com.google.gson.annotations.SerializedName;

/**
 * GitHub contributor
 */
public class Contributor {
    public String login;
    public int contributions;
    @SerializedName("avatar_url")
    public String avatarUrl;
}

File: app/src/main/java/com/jawnnypoo/openmeh/github/Contributor.kt
package com.jawnnypoo.openmeh.github

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * GitHub contributor
 */
@Serializable
data class Contributor(
    @SerialName("avatar_url")
    var avatarUrl: String? = null
)
